### PR TITLE
Improved using regedit.exe for adding a license

### PR DESCRIPTION
**Pull request by adamralph**

### CODE REVIEW

**Comment by adamralph** [comment]
@Particular/documentation how do these changes look?

Most of it is cosmetic, the only small sticking point I had when following the current docs was

> Create a `REG_MULTI_SZ` value called `License`

In the UI this is 'Multi-String Value' so it wasn't immediately obvious.

File: nservicebus/licensing/license-management.md [hunk 1]
```diff
@@ -29,13 +29,14 @@ For 64-bit operating systems the license is written to both the 32-bit and 64-bi
...
+##### Using Registry Editor
 
-- Open the `Regedit.exe` tool 
-- Goto `HKEY_LOCAL_MACHINE\Software\ParticularSoftware` or `HKEY_CURRENT_USER\Software\ParticularSoftware` 
-- Create a `REG_MULTI_SZ` value called `License` 
-- Paste the contents of the license file you've received from Particular software. 
+- Run `regedit.exe` (usually located in `%windir%`, e.g. `C:\Windows`)
+- Go to `HKEY_LOCAL_MACHINE\Software\ParticularSoftware` or `HKEY_CURRENT_USER\Software\ParticularSoftware`
+- Create a new Multi-String Value (`REG_MULTI_SZ`) named `License`
+- Paste the contents of the license file you received from Particular software.
```
Comment (mauroservienti) [inline nservicebus/licensing/license-management.md]: `Particular software` or `Particular Software` with capital S?

Comment (adamralph) [inline nservicebus/licensing/license-management.md]: Good catch! Should be capital.

Comment (adamralph) [inline nservicebus/licensing/license-management.md]: Fixed on master.

**Pull request merged**